insights/parsers/vdsm_conf.py

```diff
@@ -1,38 +1,92 @@
 """
 VDSMConfIni - file ``/etc/vdsm/vdsm.conf``
-==========================================
+------------------------------------------
+VDSMLoggerConf - file ``/etc/vdsm/logger.conf``
+-----------------------------------------------
+"""
 
-The VDSM configuration file ``/etc/vdsm/vdsm.conf`` is in the standard 'ini'
-format and is read by the IniConfigFile parser.
+from .. import IniConfigFile, parser
+from insights.specs import Specs
 
-Sample configuration file::
 
-    [vars]
-    ssl = true
-    cpu_affinity = 1
+@parser(Specs.vdsm_conf)
+class VDSMConfIni(IniConfigFile):
+    """``VDSMConfIni`` parsers configuration file ``/etc/vdsm/vdsm.conf``
+    format and is read by the IniConfigFile parser.
 
-    [addresses]
-    management_port = 54321
-    qq = 345
+    Sample configuration file::
 
+        [vars]
+        ssl = true
+        cpu_affinity = 1
 
-Examples:
-    >>> vdsm_conf = shared[VDSMConfIni]
-    >>> 'vars' in vdsm_conf
-    True
-    >>> vdsm_conf.get('addresses', 'qq')
-    '345'
-    >>> vdsm_conf.getboolean('vars', 'ssl')
-    True
-    >>> vdsm_conf.getint('addresses', 'management_port')
-    54321
-"""
+        [addresses]
+        management_port = 54321
+        qq = 345
 
-from .. import IniConfigFile, parser
-from insights.specs import Specs
 
+    Examples:
+        >>> 'vars' in conf
+        True
+        >>> conf.get('addresses', 'qq')
+        '345'
+        >>> conf.getboolean('vars', 'ssl')
+        True
+        >>> conf.getint('addresses', 'management_port')
+        54321
 
-@parser(Specs.vdsm_conf)
-class VDSMConfIni(IniConfigFile):
-    """Class for VDSM configuration file content."""
+    """
+    pass
+
+
+@parser(Specs.vdsm_logger_conf)
+class VDSMLoggerConf(IniConfigFile):
+    """``VDSMLoggerConf`` parsers configuration file
+    ``/etc/vdsm/logger.conf`` format and is read by the IniConfigFile
+    parser.
```

Comment:
This might be clearer if you worded it like this:
```
This class parses the ``/etc/vdsm/logger.conf`` file using the IniConfigFile base parser.
```